cmd/dockerd/config.go

```diff
@@ -99,6 +99,10 @@ func installCommonConfigFlags(conf *config.Config, flags *pflag.FlagSet) error {
 	flags.StringVar(&conf.ContainerdNamespace, "containerd-namespace", daemon.ContainersNamespace, "Containerd namespace to use")
 	flags.StringVar(&conf.ContainerdPluginNamespace, "containerd-plugins-namespace", containerd.PluginNamespace, "Containerd namespace to use for plugins")
 
+	flags.StringVar(&conf.HTTPProxy, "http-proxy", "", "HTTP proxy url for outgoing traffic (example: http://user:pass@proxy.corp.example.com:8080)")
+	flags.StringVar(&conf.HTTPSProxy, "https-proxy", "", "HTTPS proxy url for outgoing traffic (example: https://user:pass@proxy.corp.example.com:8080)")
```

Comment:
should we remove the `user:pass` from the examples here? Not sure it's something we really want to _encourage_ (I know some setups use it, but it's not "ideal")

Reply:
I guess the theory was probably to loosely document the syntax for users who need authentication, but I agree with your suggestion -- we probably don't want to encourage it, and users who need it will likely try this syntax first anyhow.